erc777.k

```diff
@@ -78,11 +78,19 @@ module ERC777
   rule <k> balanceOf ( Holder:Int  ) => Balance ... </k>
        <id> Holder </id>
        <balance> Balance </balance>
+    requires Holder =/=Int 0
+
+  //Required for send/operatorSend revert cases.
+  rule balanceOf(0) => 0
```

Comment:
I'm not sure if this rule is necessary. Logically, we can assume that the `<balance>` map is defined for all addresses (even for 0x0). For testing, we can initialize the balance map to include the entry `0 |-> 0`.